src/cart/crt_iv.c

```diff
@@ -1206,9 +1252,23 @@ crt_ivf_rpc_issue(d_rank_t dest_node, crt_iv_key_t *iv_key,
 
 	input->ifi_ivns_id = ivns_internal->cii_gns.gn_ivns_id.ii_nsid;
 	input->ifi_ivns_group = ivns_internal->cii_gns.gn_ivns_id.ii_group_name;
+	/* If version passed in does not match current ivns version,	   */
+	/* then the version has changed during the rpc build process.	   */
+	/* MUST not set (could cause a race):				   */
+	/*  input->ifi_grp_ver = ivns_internal->cii_grp_priv->gp_membs_ver */
+
```

Comment:
Multi-line comments have a different structure. See beginning of the file with copyright multi-line comment example

Reply:
Have modified to match standard.